src/main/java/com/kuit/agarang/domain/home/controller/SettingController.java

```diff
@@ -0,0 +1,44 @@
+package com.kuit.agarang.domain.home.controller;
+
+import com.kuit.agarang.domain.home.model.dto.BabySettingResponse;
+import com.kuit.agarang.domain.home.model.dto.BabySettingUpdateRequest;
+import com.kuit.agarang.domain.home.model.dto.FamilySettingResponse;
+import com.kuit.agarang.domain.home.model.dto.GlobalSettingResponse;
+import com.kuit.agarang.domain.home.service.SettingService;
+import com.kuit.agarang.global.common.model.dto.BaseResponse;
+import com.kuit.agarang.global.common.model.dto.BaseResponseStatus;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/home/setting")
+@RequiredArgsConstructor
+public class SettingController {
+
+  private final SettingService settingService;
+
+  @GetMapping()
+  public ResponseEntity<BaseResponse> getSetting() {
```

Comment:
```suggestion
  public ResponseEntity<BaseResponse<GlobalSettingResponse>> getSetting() {
```
같이 넣어주면 응답값이 뭔지 명시할 수 있어요!